### PR TITLE
Update the quickstart

### PR DESCRIPTION
**Pull request by mckayward**
Proposed update to the quickstart tutorial.

This was kind of sitting stale for a bit, so I think feedback would be good. It's a fair amount longer than the old quickstart, so maybe we can get rid of some stuff if it's too tedious to get through. I am open to any feedback.

I ran through the quick-start from start to finish with the new code and descriptions and everything seems to be in line, but let me know if you see any issues.

Being served [here](mckayward.github.io/floyd-docs/getstarted/quick_start/)

### CODE REVIEW

**Comment by saiprashanths** [comment]
Starting to review the article. But, at first glance, this is no longer "quick" :) It's a pretty comprehensive run through, and there's a lot of value to it.

Can we distill down the commands (or subset of commands) from this and create an actual quick start? Example: [Hy Quickstart](https://hy.readthedocs.io/en/stable/quickstart.html). We can either put this on the top of this article or create a separate quick start tutorial and call this a more comprehensive tutorial?

File: docs/getstarted/quick_start.md [hunk 1]
```diff
@@ -1,119 +1,224 @@
...
 $ cd quick-start
-```
-
-```bash
 $ ls
 LICENSE    mnist_cnn.py    mnist_cnn.ipynb    README.md
 ```
 
-This repository contains two important files: `mnist_cnn.py` (a Python script to train a convolutional neural network model against the MNIST dataset) and `mnist_cnn.ipynb` (a Jupyter Notebook to interactively explore the MNIST dataset). 
+This repository contains a file we'll use to get started: `mnist_cnn.py`. It's
```
Comment (saiprashanths) [inline docs/getstarted/quick_start.md]: The filename will change, correct?

**Comment by mckayward** [comment]
Yeah I agree with you. I'll cut some stuff out
  
  
  

  
  
  
  
  
>

File: docs/getstarted/quick_start.md [hunk 2]
```diff
@@ -1,119 +1,224 @@
...
+
+## Get the Dataset
 
-This tells FloydHub that all the Jobs you run from this local directory belong to your Project named `quick-start`.
+To run our deep-learning script, we'll need to give it access to the MNIST
+dataset. You probably know that the MNIST dataset is actually available within
+the TensorFlow package itself, but for the purposes of this tutorial we have
+separated out the dataset so you can get a feel for what it's like to work
+with datasets on FloydHub. We have the MNIST dataset pulicly available on
+FloydHub [here]().
```
Comment (saiprashanths) [inline docs/getstarted/quick_start.md]: Missing link

File: docs/getstarted/quick_start.md [hunk 3]
```diff
@@ -1,119 +1,224 @@
...
+Specifies that we want our job run on a GPU server. More info
+on instance types [here](../../guides/run_a_job/#instance-type).
+
+### --data mckay/datasets/mnist/1:/mnist
+
+The `mnist_cnn.py` script expects the MNIST data to be located at `/mnist` on
+the computer where the script runs. We can use the `--data` flag to ensure that
+our dataset is available to our code at `/mnist`.
+
+We pass the `--data` flag the name of our dataset (`mckay/datasets/mnist/1`)
```
Comment (saiprashanths) [inline docs/getstarted/quick_start.md]: Probably link to the URL of the dataset somewhere so users can see the connection between the URL and the data name

File: docs/getstarted/quick_start.md [hunk 4]
```diff
@@ -125,23 +230,97 @@ $ floyd logs -t alice/quick-start/1
...
+The output of your code is printed in the `Run Output` section of the logs,
+between the `#########` lines. Anything you log or print in your code will
+appear here, so this is a great way to monitor the progress of your model
+training command. In our `quick-start` project, we're logging the training
+accuracy of our model.
 
 Using the `-t` (tail) flag will stream the logs as they are generated.
 
-You can also view the logs in your browser using your `Job URL`. However, the logs in the Dashboard are not currently refreshed dynamically, so you'll need to refresh your browser periodically or press `F5` to get the latest logs from the Dashboard.
+You can also view the logs in your browser using your `Job URL`. However, the
```
Comment (saiprashanths) [inline docs/getstarted/quick_start.md]: This is no longer true :) Logs auto-refreshed when job is running. You can remove this paragraph

File: docs/getstarted/quick_start.md [hunk 5]
```diff
@@ -125,23 +230,97 @@ $ floyd logs -t alice/quick-start/1
...
+
+## Storing Your Model for Future Use
+
+We want to save the model we trained so we can use it later, maybe to iterate
+on it or to check its accuracy using an evaluation script.
+
+To save something during our job that we want to save for later, we just need
+to make sure it gets saved at `/output` during our job. Anything in `/output`
+at the end of a job will be saved for us and we can reuse it later. Take a look
+at [line 108](https://github.com/floydhub/quick-start/blob/master/mnist_cnn_train.py#L108)
```
Comment (saiprashanths) [inline docs/getstarted/quick_start.md]: Insert a screenshot or a code snippet here

File: docs/getstarted/quick_start.md [hunk 6]
```diff
@@ -125,23 +230,97 @@ $ floyd logs -t alice/quick-start/1
...
+
+```
+
+$ floyd run \
+--gpu \
+--env tensorflow-1.3 \
+--data mckay/datasets/mnist/1:/mnist \
+--data alice/projects/quick-start/1/output:/model \
+'python mnist_cnn_eval.py'
+```
```
Comment (saiprashanths) [inline docs/getstarted/quick_start.md]: Show the output so the user knows what to expect (e.g. accuracy numbers)

**Comment by saiprashanths** [comment]
Read through it. Looks great - solid intro tutorial!

**Comment by saiprashanths** [comment]
> Yeah I agree with you. I'll cut some stuff out

After reading through it, I really like it. I don't think we have to cut anything out of this. It might just be worthwhile to have a true quickstart for the impatient reader, in addition to this more comprehensive walkthrough

**Comment by mckayward** [comment]
Okay I made the merge-necessary changes. Per our conversation outside this PR, I am merging now and we will iterate later with a shorter quickstart, and then put this into its own tutorial
